packages/teleport/src/Kube/fixtures/index.ts

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { Kube } from 'teleport/services/kube'
+
+export const kubes: Kube[] = [
+    {
+      name: 'tele.logicoma.dev-prod',
+      labels: [
+        { name: 'kernal', value: '4.15.0-51-generic' },
+        { name: 'env', value: 'prod' },
+      ],
+      labelTexts: [ 'kernal: 4.15.0-51-generic', 'env: prod' ],
+    },
+    {
+      name: 'tele.logicoma.dev-staging',
+      labels: [{ name: 'env', value: 'staging' }],
+      labelTexts: [ 'env: staging' ],
+    },
+    {
+      name: 'cookie',
+      labels: [
+        { name: 'cluster-name', value: 'some-cluster-name' },
+        { name: 'env', value: 'idk' },
+      ],
+      labelTexts: [ 'cluster-name: some-cluster-name', 'env: idk' ],
+    }
+];
```

Comment:
I don't think your prettier is using our config file. We should have trailing commas, and new line at end of file (this will be done automatically on save, if prettier is set up correctly).

We recently edited our config: https://github.com/gravitational/webapps/#setup-prettier-on-vscode

After configuring, go this file, and save (ctrl + s), and see if changes are made automatically.

Btw, that red symbol with minus sign means no newline (try hovering over it)

Reply:
I've applied prettier config to my VSCode